brianlong/www.validators.app
# 180586118 add average score to stake pools

#### What's this PR do?
- Adds average_score to stake pools

#### How should this be manually tested?
- Run migrations
- Run stake pools pipeline (stake_logic_demo) or deploy to staging 
- Check average_score values in stake pools in console
- Make sure there're no errors thrown by the script

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/180586118)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
